changelog.dd

```diff
@@ -175,6 +176,10 @@ $(BUGSTITLE Compiler Changes,
         }
         ---
     )
+
+    $(LI $(LNAME2 dash_safe, Add -transition=safe switch.)
+	$(P Enables enhanced @safe checking, which will break some existing code.)
```

Comment:
"@safe" should be in backquotes (anything not part of the English text should be adorned appropriately)